acouchbase/bucket.py

```diff
@@ -11,7 +11,10 @@
 
 class Bucket(AsyncBucket):
     def __init__(self, *args, **kwargs):
-        loop = asyncio.get_event_loop()
+        loop = kwargs.pop('loop', None)
```

Comment:
How about:

    def __init__(self, *args, loop, **kwargs):
        loop = loop or asyncio.get_event_loop()

This way, kwargs does not get altered and the code is a little more explicit.